### ISSUE TITLE
IMOS:   check_coordinate_variables - no fail message

### ISSUE THREAD

**Issue by lbesnard** [aodn/compliance-checker#20]
netcdf-checker output:

```
    space_time_coordinate              :3:     1/ 3 :  
        check_variable_present         :3:     1/ 3 :  
```

There is no description more than what is above, which variable is affected, what the problem is.

file:
/mnt/ebs/wip/ANMN/timeSeriesProfile_po_permission/new_tb/IMOS_ANMN-QLD_AETVZ_20130620T075902Z_ITFTIS_FV00_ITFTIS-1306-Workhorse-ADCP-464.5_END-20140111T055902Z_C-20150515T020357Z.nc

**Comment by fxmzb123** [comment]
@lbesnard and @mhidas I have created a new branch and pull request for fixing this.

**Comment by mhidas** [comment]
@fxmzb123 I think the code logic is also a bit wrong here. What we're checking is that _at least one_ coordinate variable in the file is a spatial or time coordinate. [This file](http://thredds.aodn.org.au/thredds/catalog/IMOS/eMII/checker_test/ANMN/timeSeriesProfile/catalog.html?dataset=IMOS/eMII/checker_test/ANMN/timeSeriesProfile/IMOS_ANMN-NRS_AETVZ_20140919T050000Z_NRSROT-ADCP_FV01_NRSROT-ADCP-1409-Workhorse-ADCP-44_END-20150128T030000Z_C-20150129T090249Z.nc) has the TIME coordinate, yet the check returns a fail.

Looking at the code, I think the problem is it's setting `passed = False` (line [598](https://github.com/aodn/compliance-checker/blob/IMOS-Checker/compliance_checker/imos/imos.py#L598)) even when the check has been passed in a previous iteration.

**Comment by mhidas** [comment]
... and the same goes for `reasoning = None` on the next line. That is why we're not getting a reasoning message in the output.

**Comment by fxmzb123** [comment]
Hi @mhidas I have fixed the wrong logic, please see the pull request.

**Comment by mhidas** [comment]
Fixed via #28
